src/ApiCore/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -116,26 +112,18 @@ class OperationsGapicClient
      */
     const CODEGEN_NAME = 'gapic';
 
-    /**
-     * The code generator version, to be included in the agent header.
-     */
-    const CODEGEN_VERSION = '0.0.5';
-
     private static function getClientDefaults()
     {
         return [
             'serviceName' => self::SERVICE_NAME,
-            'descriptorsConfigPath' => __DIR__.'/../resources/operations_descriptor_config.php',
             'clientConfig' => __DIR__.'/../resources/operations_client_config.json',
-            'disableRetries' => false,
-            'auth' => null,
-            'authConfig' => null,
-            'transport' => null,
+            'descriptorsConfigPath' => __DIR__.'/../resources/operations_descriptor_config.php',
+            'credentialsConfig' => [
+            ],
             'transportConfig' => [
-                'rest' => [
-                    'restClientConfigPath' => __DIR__.'/../resources/operations_rest_client_config.php',
-                ]
+                'rest' => ['restClientConfigPath' => __DIR__.'/../resources/operations_rest_client_config.php'],
             ],
+            'versionFile' => __DIR__.'/../../../VERSION',
```

Comment:
Hmm, this may only apply for the GAPICs living in google-cloud-php. Would we want to omit this here?

Reply:
I think there is also a problem that can occur with the path being incorrect for deeper nested generated code. I have filed https://github.com/googleapis/gapic-generator/issues/1978, will fix manually for now.